starter-workflows/code-scanning/frogbot-scan-pr.yml

```diff
@@ -35,23 +39,9 @@ jobs:
       #    with:
       #      node-version: "16.x"
 
-      # - uses: gradle/gradle-build-action@v2
+      # The full template list can be found at https://github.com/jfrog/frogbot/tree/master/templates/github-actions/scan-pull-request
```

Comment:
The full template list... --> The full template list with the required GitHub Actions...

Reply:
done